### PR TITLE
Style CO Petitions Duplicate Check (CO-1796)

### PR DESCRIPTION
**Pull request by spartanicus**
This PR formats and styles the CO Petition Duplicate Check against a Match server to look like the following:
image

### CODE REVIEW

File: app/Lib/lang.php [hunk 1]
```diff
@@ -134,6 +134,7 @@
   'ct.co_people.pl' =>          'CO People',
   'ct.co_person_roles.1' =>     'CO Person Role',
   'ct.co_person_roles.pl' =>    'CO Person Roles',
+  'ct.co_petition.duplicate.check' => 'CO Petition Duplicate Check',
```
Comment (boshrin) [inline app/Lib/lang.php:137]: This isn't a controller localization, this key should be elsewhere, probably `op.`.

File: app/Lib/lang.php [hunk 2]
```diff
@@ -1672,6 +1677,7 @@
   'fd.name.primary_name' => 'Primary',
   'fd.name.fields.req' => 'A name must consist of at least these fields:',
   'fd.nested' => 'Nested',
+  'fd.new' =>           'New',
```
Comment (boshrin) [inline app/Lib/lang.php:1680]: `fd.match.new`?

File: app/Lib/lang.php [hunk 3]
```diff
@@ -2064,6 +2070,7 @@
   'op.add' =>         'Add',
   'op.add-a' =>       'Add %1$s',
   'op.add.new' =>     'Add a New %1$s',
+  'op.add.new.literal' => 'Add new',
```
Comment (boshrin) [inline app/Lib/lang.php:2073]: `op.match.new`?

File: app/Lib/lang.php [hunk 4]
```diff
@@ -2186,7 +2193,10 @@
   'op.manage' =>      'Manage',
   'op.manage-a' =>    'Manage %1$s',
   'op.manage.grm' =>  'Manage Group Memberships',
+  'op.match.add.confirm' => 'Are you sure you wish to add a new CO Person record?',
```
Comment (boshrin) [inline app/Lib/lang.php:2196]: "Are you sure you want to create a new CO Person record"?

File: app/Lib/lang.php [hunk 5]
```diff
@@ -2186,7 +2193,10 @@
...
+  'op.match.merge' => 'Merge',
   'op.match.new' =>   'Assign a new Reference Identifier',
+  'op.match.select.confirm' => 'Do you want to merge the new identity with this existing identity?',
```
Comment (boshrin) [inline app/Lib/lang.php:2199]: "Are you sure you want too..."?

**Review by boshrin — CHANGES_REQUESTED** [review]
Proposed some wordsmithing.

**Comment by spartanicus** [comment]
Thanks Benn - the text changes have been put in place.

File: app/Lib/lang.php (hunk 1 above)
Comment (spartanicus) [reply app/Lib/lang.php]: The lang keys have been reworked into op.match and fd.match. 
This one is "op.match.duplicate.check"

File: app/Lib/lang.php (hunk 2 above)
Comment (spartanicus) [reply app/Lib/lang.php]: Changed as suggested.

File: app/Lib/lang.php (hunk 4 above)
Comment (spartanicus) [reply app/Lib/lang.php]: Changed as suggested.

File: app/Lib/lang.php (hunk 5 above)
Comment (spartanicus) [reply app/Lib/lang.php]: Changed as suggested.

**Pull request merged**